src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax.model;
+
+public class FruitTransaction {
+    private Operation operation;
+    private Fruit fruit;
+    private int quantity;
+
+    public FruitTransaction(Operation operation, Fruit fruit, int quantity) {
+        this.operation = operation;
+        this.fruit = fruit;
+        this.quantity = quantity;
+    }
+
+    public Operation getOperation() {
+        return operation;
+    }
+
+    public void setOperation(Operation operation) {
```

Comment:
If you do not plan to modify fields it's better to remove setter methods and make fields final.